app/agent/views/check_your_answers.scala.html

```diff
@@ -92,6 +93,14 @@
     summaryRow(rowName, rowQuestion, rowAnswer, rowUrl)
 }
 
+@accountingMethodProperty(income: AccountingMethodPropertyModel) = @{
+    val rowName = AccountingMethodPropertyId
+    val rowQuestion = messages("agent.summary.income_type_property")
+    val rowAnswer = accountingMethodText(income)
+    val rowUrl = agent.controllers.business.routes.PropertyAccountingMethodController.show(editMode = true).url
+    summaryRow(rowName, rowQuestion, rowAnswer, rowUrl)
+}
+
 @core.views.html.templates.main_template(title = Messages("agent.summary.title")) {
```

Comment:
You have imported core.views.html.templates.main_template, so this can be simplified
```suggestion
@main_template(title = Messages("agent.summary.title")) {
```